src/vs/workbench/browser/parts/titlebar/media/titlebarpart.css

```diff
@@ -8,24 +8,133 @@
 	width: 100%;
 	font-size: 12px;
 	padding: 0 70px;
-	overflow: hidden;
 	flex-shrink: 0;
 	align-items: center;
 	justify-content: center;
 	user-select: none;
 	-webkit-app-region: drag;
 	zoom: 1; /* prevent zooming */
-	line-height: 22px;
 	height: 22px;
 	display: flex;
+	order: 1;
 }
 
 .monaco-workbench > .part.titlebar > .window-title {
 	flex: 0 1 auto;
 	overflow: hidden;
 	white-space: nowrap;
-	line-height: 22px;
 	text-overflow: ellipsis;
-	-webkit-app-region: drag;
 	zoom: 1; /* prevent zooming */
+}
+
+/***********************\
+| Windows and Linux CSS |
+\***********************/
+
+.monaco-workbench.windows > .part.titlebar, .monaco-workbench.linux > .part.titlebar {
```

Comment:
@sbatten I think it is easier to read multiple CSS rules that are separated with "`,`" by putting each rule on a separate line